### ISSUE TITLE
Uncaught DOMException: Failed to execute 'querySelector' on 'Element'

### ISSUE THREAD

**Issue by creage** [naver/billboard.js#1206]
## Description
Some strings used as data key names lead to DOMException.

## Steps to check or reproduce
Have a simple Donut chart, but with some tricky names
```javascript
var chart = bb.generate({
  data: {
    columns: [
	["SQ==", 1],
	["Tg==", 1],
	["U0tBQg==", 4]
    ],
    names: {
        'SQ==': "I",
        'Tg==': "N",
        'U0tBQg==': "S"
    },
    type: "donut"  
  },
  bindto: "#donutChart"
});
```
Render the chart (you can use [Examples ](https://naver.github.io/billboard.js/demo/#Chart.DonutChart)page), open browser console and try to hover over donut piece. See errors thrown.

P.S. The error was found after trying to workaround #1198 , using standard ``btoa()`` methods.

One of the approaches could be to [sanitize CSS selectors](https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape) by using a [standard polyfill](https://github.com/mathiasbynens/CSS.escape/blob/master/css.escape.js) function.

**Comment by netil** [comment]
I tested the example code, but I'm not getting any errors on hovering. (tested on Chrome/FF)

**Comment by creage** [comment]
@netil hmm, very strange. It fails for with this error

![image](https://user-images.githubusercontent.com/910967/72416316-b45ec600-3776-11ea-9903-d2ccae2e1b8f.png)

**Comment by netil** [comment]
@creage, got it. This issue was already fixed by #1173.
I couldn't get the error, because I was testing with the fixed build :)

Try with the `nightly` build.
- https://github.com/naver/billboard.js#nightly-version
